internal/providers/terraform/aws/testdata/mwaa_environment_test/mwaa_environment_test.tf

```diff
@@ -0,0 +1,84 @@
+provider "aws" {
+  region                      = "us-east-1"
+  skip_credentials_validation = true
+  skip_metadata_api_check     = true
+  skip_requesting_account_id  = true
+  skip_get_ec2_platforms      = true
+  skip_region_validation      = true
+  access_key                  = "mock_access_key"
+  secret_key                  = "mock_secret_key"
+}
+
+resource "aws_iam_role" "r" {
+  name = "awsconfig-example"
+
+  assume_role_policy = <<POLICY
+{}
+POLICY
+}
+
+resource "aws_security_group" "example" {
+}
+
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "aws_subnet" "private" {
+  vpc_id     = aws_vpc.main.id
+  cidr_block = "10.0.1.0/24"
+
+  tags = {
+    Name = "Main"
+  }
+}
+
+resource "aws_s3_bucket" "example" {
+  bucket = "mybucket"
+  acl    = "private"
+
+  tags = {
+    Name = "My bucket"
+  }
+}
```

Comment:
seems to be a mix of this and hardcoding fake data in `testdata`:
https://github.com/infracost/infracost/blob/4e318feff1c6c9ccf37eb388835dd75bb1a20a0d/internal/providers/terraform/aws/testdata/directory_service_directory_test/directory_service_directory_test.tf#L19
Don't think we're planning on actually deploying, so I think it'd be quicker/easier to just hardcode fake data for these:
https://github.com/infracost/infracost/pull/1123